src/Akeneo/Connectivity/Connection/front/src/connect/components/ConnectedApp/connectedAppScopesExplorer.ts

```diff
@@ -0,0 +1,15 @@
+import {ConnectedApp} from '../../../model/Apps/connected-app';
+
+const containsProductScope: (connectedApp: ConnectedApp) => boolean = (connectedApp: ConnectedApp): boolean => {
+    const productsRegexp = new RegExp('.*products.*');
+
+    return connectedApp.scopes.find(scope => productsRegexp.test(scope)) !== undefined;
```

Comment:
not a fan of regex TBH I would rather have an explicit list
```suggestion
    const productScopes = ['read_products', 'write_products', 'delete_products');

    return connectedApp.scopes.find(scope => productScopes.inlcudes(scope)) !== undefined;
```
WDYT ?